content/millennium/r4/scheduling/appointment.md

```diff
@@ -316,6 +316,7 @@ _Implementation Notes_
   * `Appointment.participant` must be a list containing exactly one Patient participant and at least one Location participant.
   * `Appointment.participant.actor` must be a reference to a Patient or a Location.
   * `Appointment.participant.status` must be set to `needs-action`.
+  * `Appointment.reasonCode` must be a list containing exactly one CodeableConcept.
```

Comment:
this applies to both proposed and booked so move it up a level

we can say 'Appointment.reasonCode, if set, must be ...'

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/commit/063202ecf7b52c89b017b9b8d20412808d417975#diff-1d7df8ceb9f0ae8471323516e037c662R314)